src/components/RadioInputElement.jsx

```diff
@@ -1,8 +1,14 @@
-const RadioInputElement = ({ children, id, value }) => {
+const RadioInputElement = ({ label, id, value, required }) => {
 	return (
 		<>
-			<input type="radio" name="purchase-date" id={id} value={value} required />
-			<label htmlFor={id}>{children}</label>
+			<input
+				type="radio"
+				name="purchase-date"
+				id={id}
+				value={value}
+				required={required}
+			/>
+			<label htmlFor={id}>{label}</label>
 			<br />
```

Comment:
RadioInputElement takes a label prop instead of children prop, as well as required prop